examples/beam/src/main/java/org/apache/nemo/examples/beam/MultinomialLogisticRegression.java

```diff
@@ -225,7 +227,7 @@ public void processElement(final ProcessContext c) throws Exception {
     @FinishBundle
     public void finishBundle(final FinishBundleContext c) {
       for (Integer i = 0; i < gradients.size(); i++) {
-        c.output(KV.of(i, gradients.get(i)), null, null);
+        c.output(KV.of(i, gradients.get(i)), BoundedWindow.TIMESTAMP_MIN_VALUE, GlobalWindow.INSTANCE);
```

Comment:
Please add a note here that this enforces a global window (batching), where all data elements of the corresponding PCollection are grouped and emitted downstream together.